package.json

```diff
@@ -2,18 +2,19 @@
   "name": "budgetkey-app-main-page",
   "version": "1.0.0",
   "description": "BudgetKey Main Page App",
+  "engines": {
+    "node": "^8.1.0",
+    "npm": "^5.2.0"
+  },
   "scripts": {
-    "start": "tsc && concurrently \"tsc -w\" \"lite-server -c bs-server.js\" ",
-    "e2e": "tsc && concurrently \"http-server -s\" \"protractor protractor.config.js\" --kill-others --success first",
-    "lint": "tslint ./app/**/*.ts -t verbose",
-    "lite": "lite-server",
+    "start": "http-server ./dist -p 8000",
     "pree2e": "webdriver-manager update",
-    "test": "tsc && concurrently \"tsc -w\" \"karma start karma.conf.js\"",
-    "test-once": "tsc && karma start karma.conf.js --single-run",
```

Comment:
I believe that this is being used in the travis-ci script.
I enabled Travis now so that we can make sure it passes.